corehq/ex-submodules/casexml/apps/phone/dbaccessors/sync_logs_by_user.py

```diff
@@ -22,3 +27,39 @@ def get_all_sync_logs_docs():
         reduce=False,
     )]
     return iter_docs(SyncLog.get_db(), all_sync_log_ids)
+
+
+def get_sync_logs_for_user(user_id, limit):
+    rows = synclog_view(
+        "phone/sync_logs_by_user",
+        startkey=[user_id, {}],
+        endkey=[user_id],
+        descending=True,
+        reduce=False,
+        limit=limit,
+        include_docs=True,
+    )
+    sync_log_jsons = (row['doc'] for row in rows)
+    return [properly_wrap_sync_log(sync_log_json) for sync_log_json in sync_log_jsons]
+
+
+def synclog_view(view_name, **params):
+    return combine_views([SyncLog.get_db(), get_db(None)], view_name, **params)
+
+
+def combine_views(dbs, view_name, **params):
```

Comment:
we have [a similar set of functionality in dimagi-utils](https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/couch/pagination.py#L31) that also supports `skip` (albeit hackily).

this is much simpler so seems good, just noting